### ISSUE TITLE
API: meglévő fordítások listája

### ISSUE THREAD

**Issue by gyuris** [borazslo/szentiras.hu#130]
Egy dolgot hiányolok az [API](http://szentiras.hu/api)-ból: Nem lehet korrekt módon lekérdezni a meglévő fordításokat. Jó lenne, ha JSON választ tudnátok adni egy ilyen kérésre pl. a  `http://szentiras.hu/api/forditasok` címen. Valami olyasmire gondolok, amiben a `name`, `abbrev`, `denom`, `lang`, `copyright`, `publisher`, `publisher_url` mezőket adnátok vissza minden adatbázisban lévő fordításról.

Indoklás: 
1. Azt gondolom, hogy ez hasznos lenne, hisz számos hívásban tudni kell a fordítás rövidítését, amit jelenleg úgy kell lenézni a [weboldalból](http://szentiras.hu/forditasok).
2. Az API tervezett képet mutatna :-).
3. Ezzel én pl. el tudnám kerülni a fordításaitok [bedrótozását a saját kódomba](https://cgit.drupalcode.org/szentirashu_formatter/tree/src/Plugin/Field/FieldFormatter/SzentirashuFormatter.php#n43), mivel akkor ezt is dinamikusan lehetne lekérdezni. Vagyis kevesebb munka lenne akkor, ha megváltoztatnátok a fordítások azonosítóit, vagy újat vinnétek fel.

Tudom, hogy a minimálisan szükséges adat pl. kinyerhető ebből a dokumentációban is szereplő [hívásból](http://szentiras.hu/api/forditasok/10100100100), de ez így nem elegáns.

**Comment by gyuris** [comment]
Az új http://ujszov.hu az oldalpanelen lévő [allTranslations](https://github.com/briff/ujszov.szentiras.hu/issues/25) boxban honnan tudja, hogy pl. a régi protestáns fordítás már nem él? Ez ott is kézi beállítás?

**Comment by briff** [comment]
Igen, kézi beállítás. Az adatbázisból még nem szedtük ki teljesen a régi protestáns fordítást (részben abból a megfontolásból, hogy van, aki fejből tud valami szöveget és az alapján keres).

**Comment by gyuris** [comment]
Lesz ebből valami?

**Comment by briff** [comment]
https://[regi.szentiras.hu/api/ref/Mt1,1/*](https://regi.szentiras.hu/api/ref/Mt1,1/*) Tulajdonképpen ezt tudja az api :)

**Comment by gyuris** [comment]
Köszönöm, hogy ezt elkészítettétek. 
A kis Drupal [igehely hivatkozásformázó modulomat](https://szentiras.eu/api/forditasok) át is állítottam a szentiras.eu-ra, valamint a fordítások listája sincs már beégetve, hanem innen az API-ból kéri le azokat dinamikusan: https://szentiras.eu/api/forditasok
